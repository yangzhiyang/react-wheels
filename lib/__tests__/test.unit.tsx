import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../button';

describe('button test', () => {
  it('是个div', () => {
    const tree = renderer.create(<Button />).toJSON();
    expect(tree).toMatchSnapshot();
  });
})