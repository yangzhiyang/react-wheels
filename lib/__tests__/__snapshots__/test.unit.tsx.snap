// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button test 是个div 1`] = `
<div>
  按钮
</div>
`;
